docs/quick-start/index.md

```diff
@@ -682,19 +685,240 @@ public Server(String serverName) {
+
+### Closing the client
+
+The `close()` method simply delegates to the method of the `io.spine.client.Client` class.
+We also need to tell the calling code if the client has finished its job. 
+This is what the `isDone()` method for:
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="boolean isDone()" 
+             end="    }"?>
+```java
+public boolean isDone() {
+    return client.subscriptions()
+                 .isEmpty();
+}
+```
+The method checks active subscriptions. If there are none, we got the event and cleared its
+subscription.
+
+Now, let's put it all together.
+
+## Orchestrating the example
+
+Let's review the `Example` class and its `main()` method. It simulates client-server communication
+scenario.
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/Example.java" 
+             start="void main(" 
+             end="    }"?>
+```java
+public static void main(String[] args) {
+    String serverName = UUID.randomUUID().toString();
+    Server server = new Server(serverName);
+    Client client = null;
+    try {
+        server.start();
+        client = new Client(serverName);
+        client.sendCommand();
+        while (!client.isDone()) {
+            sleepUninterruptibly(Duration.ofMillis(100));
+        }
+```
+The first thing the method does is creating a `Server` using a random name.
+
+Then, the server is started. It is done in the `try/catch/finally` block because the `start()` 
+method may throw `IOException` in case of communication problems reported by gRPC. 
+The `catch` block calls the `onError()` method of the `Example` class which simply prints
+the exception. A real-world application would use more sophisticated exception handling.
+ 
+After the `Server` is started, we create a `Client` using the name of the server generated before.
+Then, the client sends the command, and we wait until the client finishes its job using
+the statically imported method `sleepUninterruptibly()` provided by Guava.
+
+The `main()` method finishes by closing the client and shutting down the server.  
+
+## Summary
+
+Although the “business case” of this example is trivial, it shows basic steps of development 
+of a solution based on Spine Event Engine framework. 
+
+The development starts with the discovery of the business using EventStorming or another
+learning approach.
+
+Then, we select a Bounded Context and define events, commands, Value Objects and states of entities
+using Protobuf. Using these `.proto` files Spine Model Compiler generates the code implementing
+the defined data types.
+
+After that we add the business logic for handling commands or events in entity classes derived from
+`Aggregate`, `ProcessManager`, or `Projection`. 
+
+Then, the selected Bounded Context is assembled using these entity types and tested using
+the black-box approach. A test suite sends signals (e.g. commands or events) to the implementation
```

Comment:
Suggestion: "Then, these entity types are assembled into a Bounded Context and tested as a whole." I'd alse skip the details on testing entierly for now (there's no mention of them in the text anyway). The testing subject is very needed and should be covered in a separate document/guide.

Reply:
Good idea!